Clockify.Net/ClockifyClient.cs

```diff
@@ -386,6 +386,44 @@ public Task<IRestResponse> DeleteTimeEntryAsync(string workspaceId, string timeE
 		/// Get templates for current user on specified workspace. See Clockify docs for query params explanation.
 		/// </summary>
 		public Task<IRestResponse<List<TimeEntryDtoImpl>>> FindAllTimeEntriesForUserAsync(
+			string workspaceId,
+			string userId,
+			string description = null,
+			DateTimeOffset? start = null,
+			DateTimeOffset? end = null,
+			string project = null,
+			string task = null,
+			bool? projectRequired = null,
+			bool? taskRequired = null,
+			bool? considerDurationFormat = null,
+			bool? hydrated = null,
+			bool? inProgress = null,
+			int page = 1,
+			int pageSize = 50)
+		{
+			var request = new RestRequest($"workspaces/{workspaceId}/user/{userId}/time-entries");
+
+			if (description != null) { request.AddQueryParameter(nameof(description), description); }
+			if (start != null) { request.AddQueryParameter(nameof(start), start.Value.ToString("yyyy-MM-ddTHH:mm:ssZ")); }
+			if (end != null) { request.AddQueryParameter(nameof(end), end.Value.ToString("yyyy-MM-ddTHH:mm:ssZ")); }
+			if (project != null) { request.AddQueryParameter(nameof(project), project); }
+			if (task != null) { request.AddQueryParameter(nameof(task), task); }
+			if (projectRequired != null) { request.AddQueryParameter("project-required", projectRequired.ToString()); }
+			if (taskRequired != null) { request.AddQueryParameter("task-required", taskRequired.ToString()); }
+			if (considerDurationFormat != null) { request.AddQueryParameter("consider-duration-format", considerDurationFormat.ToString()); }
+			if (hydrated != null) { request.AddQueryParameter(nameof(hydrated), hydrated.ToString()); }
+			if (inProgress != null) { request.AddQueryParameter("in-progress", inProgress.ToString()); }
+
+			request.AddQueryParameter(nameof(page), page.ToString());
+			request.AddQueryParameter("page-size", pageSize.ToString());
+
+			return _client.ExecuteGetAsync<List<TimeEntryDtoImpl>>(request);
+		}
+		
+		/// <summary>
+		/// Get hydrated time entries for current user on specified workspace. See Clockify docs for query params explanation.
+		/// </summary>
+		public Task<IRestResponse<List<HydratedTimeEntryDtoImpl>>> FindAllHydratedTimeEntriesForUserAsync(
```

Comment:
Here too :)

Reply:
Sorry, I am having trouble with my IDE, it should be fixed now